### PR TITLE
fix: in error messages and preload boxes

### PR DESCRIPTION
**Pull request by GusBaamonde**
errors-and-preloads

Related to: https://github.com/FromDoppler/Doppler-UI-System/pull/213

### CODE REVIEW

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ const Reports = ({ dependencies: { datahubClient } }) => {
                     dateTo={state.dateTo}
                     dateFrom={state.dateFrom}
                     today={state.dailyView}
-                    emailFilter="with_email"
+                    emailFilter={'with_email'}
```
Comment (andresmoschini) [inline src/components/Reports/Reports.js]: What is this?

File: public/index.html [hunk 2]
```diff
@@ -10,13 +10,12 @@
...
-
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link
-      rel="stylesheet"
-      type="text/css"
-      href="https://cdn.fromdoppler.com/doppler-ui-library/v3.23.6/css/styles.css"
-    />
+    rel="stylesheet"
+    type="text/css"
+    href="https://cdn.fromdoppler.com/doppler-ui-library/build653/css/styles.css"
```
Comment (andresmoschini) [inline public/index.html]: Remember to update to an _official_ version before merging.

**Comment by andresmoschini** [comment]
👍

File: src/components/Reports/Reports.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/Reports.js]: It's for show the box with the data with email or without email

Comment (cbernat) [reply src/components/Reports/Reports.js]: Yes but it shouldn't  be part of this PR.

Comment (cbernat) [reply src/components/Reports/Reports.js]: I've talked this with @GusBaamonde and he did this change to be consistent with the other component, `without_email`

**Comment by GusBaamonde** [comment]
test this please

**Pull request merged**